src/scss/components/dropdown-select/mixins/SelectControl.scss

```diff
@@ -0,0 +1,63 @@
+// Mixins to style a native <select> control.
+
+@mixin __selectControl($focused: false) {
+  @if $focused {
+    &:-moz-reset-focusring {
```

Comment:
Incorrect pseudo element. Please change to `&:-moz-focusring`

Reply:
Changes have been done, please review.